crates/oxc_minifier/src/ast_passes/peephole_substitute_alternate_syntax.rs

```diff
@@ -121,6 +121,26 @@ impl<'a> Traverse<'a> for PeepholeSubstituteAlternateSyntax {
                     self.changed = true;
                 }
             }
+            // `() => { return foo })` -> `() => foo`
+            Expression::ArrowFunctionExpression(arrow_expr) => {
+                if !arrow_expr.expression
+                    && arrow_expr.body.directives.is_empty()
+                    && arrow_expr.body.statements.len() == 1
+                {
+                    if let Some(body) = arrow_expr.body.statements.first_mut() {
+                        let result = if let Statement::ReturnStatement(ret_stmt) = body {
+                            ret_stmt.argument.as_mut().map(|arg| ctx.ast.move_expression(arg))
```

Comment:
This may move the code without setting changed to true?

Reply:
don't think so but i improved this slightly to make it more readable/clear